go/glide/lib.h

```diff
@@ -0,0 +1,144 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+enum ErrorType {
```

Comment:
@Yury-Fridlyand why do you suggest the underscore? With cgo, enums are accessed by prefixing the C symbol with "enum_" already, so this would be accessed via `C.enum_ErrorType` in Go and it will not conflict with a Go type called ErrorType if we add it. If we have an extra underscore here then there would be an extra underscore to access it from Go as well, eg `C.enum__ErrorType`

Reply:
A C compiler won't compile such header I suppose due to naming conflict.
See example
https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/ne-processthreadsapi-process_information_class

Reply:
What would the naming conflict be between? The ErrorType symbol here and what else? As mentioned, I don't think it will conflict with any Go symbols

Reply:
`ErrorType` is enum and typedef
Well, if this file is autogenerated, we have to keep this as is.